controllers/managedocs_controller.go

```diff
@@ -106,6 +106,12 @@ const (
 	convergedDeploymentType                = "converged"
 	consumerDeploymentType                 = "consumer"
 	providerDeploymentType                 = "provider"
+	prometheusRemoteWriteSecretName        = "managed-ocs-prom-remote-write-secret"
+	prometheusRemoteWriteSecretKey         = "prom-remote-write-config-secret"
+	prometheusRemoteWriteIDKey             = "prom-remote-write-config-id"
+	promRemoteWriteRelabelConfigMapName    = "prom-remote-write-relabel-config"
```

Comment:
> Advantages:
Ability to change configs in-flight
- in the final deployement/in customer env will we be changing the config or is it only for development purposes
- basically I'm looking at who'll be making the changes and when if needed

Reply:
These changes would be made by devs downstream or MT-SRE in the addon manifest.
Customers should not have access to these else the addon-parameters field would have been a good place.
It is not possible to hide default parameters from the ocm ui yet!